requirements/adapter.txt

```diff
@@ -13,9 +13,14 @@ fastapi>=0.70.0,<1
 Flask>=1,<4
 Werkzeug>=2,<4
 pyramid>=1,<3
+
+# Sanic and its dependencies
+# Note: Sanic imports tracerite with wild card versions
+tracerite<1.1.2; python_version<="3.8"  # older versions of python are not compatible with tracerite>1.1.2
```

Comment:
[tracerite](https://pypi.org/project/tracerite/1.1.2/) does not specify which versions of python it supports, in [version 1.1.2](https://pypi.org/project/tracerite/1.1.2/) the project invokes `from importlib.resources import files` which is only supported as of python 3.9. tracerite>=1.1.2 therefore requires: Python >=3.9

`sanic` uses a [wildcard](https://github.com/sanic-org/sanic/blob/2beeeee49bbae16532965fb30611b6ff6a5341f3/setup.py#L115) to import tracerite ("tracerite>=1.0.0"), which make pip install the latest version of tracerite even though it is incompatible with python 3.8

To solve this we first install `tracerite<1.1.2` to satisfy sonics requirement of `tracerite>=1.0.0`

Reply:
Wildness with a wildcard but these are helpful comments! 🙏